server/src/main/java/com/cloud/user/AccountManagerImpl.java

```diff
@@ -2680,6 +2680,9 @@ public void buildACLSearchParameters(Account caller, Long id, String accountName
                         }
                     } else {
                         domainIdRecursiveListProject.third(Project.ListProjectResourcesCriteria.ListProjectResourcesOnly);
+                        if (listAll) {
```

Comment:
@Pearl1594 can you add a check here that listall is true and the user is an admin (not a domain admin, not a user but an admin)